### PR TITLE
Python Package

### PR DESCRIPTION
**Pull request by davidgasquez**
Hey there @michael-erasmus! Today I spent more time iterating on the idea of publishing the Python package automatically. 

With this PR we should be able to run `make package-python` to produce a `tar.gz` file ready to be uploaded to the releases Github tab. This process could be automated but it's a bit tricky with 2FA without using service keys. Alternatively, I thought we could just have sub-repositories and we wouldn't need to create and upload the `tar.gz` file, just pushing the changes in each repo 😄 

PS: In the future I think all of this could be automated in a sort of CI/CD pipeline. Will 🔍 a bit more soon!

### CODE REVIEW

**Comment by davidgasquez** [comment]
Also, I'm not sure if we should track the autogenerated files. 😅 

I think I'll remove them as they don't seem extremely valuable and will cause some noise in the PR diffs.

File: scripts/package-python.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Compile entities
+docker run --rm -v $PWD:$PWD -u 1000 -w $PWD \
+    znly/protoc \
+    --python_out=$PYTHON_DIR \
+    -I. $ENTITIES
+
+# Compile services
+docker run --rm -v $PWD:$PWD -u 1000 -w $PWD \
+    bufferapp/grpc-docker-python:latest \
```
Comment (davidgasquez) [inline scripts/package-python.sh]: [Would love to switch to the official image if they include Python 3.6](https://github.com/grpc/grpc-docker-library/issues/45)!

Comment (michael-erasmus) [inline scripts/package-python.sh]: 💯 Thanks for creating the issue, happy to switch to the official image if they support 3.6 :D

**Comment by michael-erasmus** [comment]
Hey @davidgasquez! Just wanted to mention that I [implemented](https://github.com/bufferapp/buda-events-consumer/commit/75f8fad3b8156178c3de9e043c5c7b4f43260d76) this in the buda-events-consumer repo, and it was super smooth!

**Pull request merged**